pkg/meta/interface.go

```diff
@@ -307,7 +307,9 @@ type Meta interface {
 	// Compact all the chunks by merge small slices together
 	CompactAll(ctx Context) syscall.Errno
 	// ListSlices returns all slices used by all files.
-	ListSlices(ctx Context, slices *[]Slice, delete bool, showProgress func()) syscall.Errno
+	ListSlices(ctx Context, slices map[Ino][]Slice, delete bool, showProgress func()) syscall.Errno
+	// GetPath returns full path of an inode; a random one is picked if it has multiple hard links
+	GetPath(ctx Context, inode Ino) (string, syscall.Errno)
```

Comment:
nit: GetPath could be implemented use  `Lookup(ctx, inode, "..")` and `Readdir(ctx, inode, 0)` without introducing a new API.

Reply:
But `Lookup(ctx, inode, "..")` does not work on files ...

Reply:
Just use `GetAttr` for now